homeassistant/components/vodafone_station/config_flow.py

```diff
@@ -133,3 +141,31 @@ async def async_step_reauth_confirm(
             data_schema=STEP_REAUTH_DATA_SCHEMA,
             errors=errors,
         )
+
+
+class VodafoneStationOptionsFlowHandler(OptionsFlow):
+    """Handle a option flow."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
```

Comment:
```suggestion
class VodafoneStationOptionsFlowHandler(OptionsFlowWithConfigEntry):
    """Handle a option flow."""
```

Reply:
Thx, was not aware of `OptionsFlowWithConfigEntry`